### PR TITLE
Erica's Chapter 5 exercises

### PR DESCRIPTION
**Pull request by ericabrody**

### CODE REVIEW

File: _posts/ericabrody/2016-02-03-Erica-Brody-Ch5-Exercises.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+---
+
+Exercise 1: Write a program which repeatedly reads numbers until the user enters “done”. Once “done” is entered, print out the total, count, and average of the numbers. If the user enters anything other than a number, detect their mistake using try and except and print an error message and skip to the next number.
+<iframe src="https://trinket.io/embed/python/e0f9320136" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Exercise 2: Write another program that prompts for a series of numbers as above and at the end prints out the maximum, minimum and average of the numbers.
+<iframe src="https://trinket.io/embed/python/fee86a045b" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Reflection:
+I know that these exercises aren't supposed to take long - as a beginning programmer, I spent about 90 minutes total figuring these out.
```
Comment (eah13) [inline _posts/ericabrody/2016-02-03-Erica-Brody-Ch5-Exercises.md:14]: Everyone's learning journey is different, so there's no 'expected time' to complete any exercise.  I'm glad you took the time to get your head around the problems.

File: _posts/ericabrody/2016-02-03-Erica-Brody-Ch5-Exercises.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+It was helpful to use scrap paper to write out the different mini steps that had to be accomplished to reach the goal. And the best part,
```
Comment (eah13) [inline _posts/ericabrody/2016-02-03-Erica-Brody-Ch5-Exercises.md:15]: You invented a problem solving strategy!  This is called pseudocode and is a very helpful way of approaching a programing problem.

File: _posts/ericabrody/2016-02-03-Erica-Brody-Ch5-Exercises.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+I found one of my errors funny today. While working on generating the minimum number in the list input by the users, the output
+kept saying "Done" or "d" - I realized that I was calculating the minimum after I checked for the word Done in the output - so the 
+computer was getting the minium of "done". In this case, output was very helpful in figuring out where the error was.
```
Comment (eah13) [inline _posts/ericabrody/2016-02-03-Erica-Brody-Ch5-Exercises.md:18]: Nice!

File: _posts/ericabrody/2016-02-03-Erica-Brody-Ch5-Exercises.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+author: ericabrody
+title: "Erica's Chapter 3 Exercises"
+---
+
+Exercise 1: Write a program which repeatedly reads numbers until the user enters “done”. Once “done” is entered, print out the total, count, and average of the numbers. If the user enters anything other than a number, detect their mistake using try and except and print an error message and skip to the next number.
+<iframe src="https://trinket.io/embed/python/e0f9320136" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
```
Comment (eah13) [inline _posts/ericabrody/2016-02-03-Erica-Brody-Ch5-Exercises.md:8]: Overall, this program is great.  One edge case to consider: if the user presses 'enter' without inputting anything the averages etc become `nan`, which stands for not a number.  This is a case where `try` `except` has failed you because `float()` doesn't throw an actual error.  You could potentially test if the user input was nothing and `continue`.

That said, there will always be edge cases :)

**Comment by eah13** [comment]
:watch:
